.github/workflows/conventions.yml

```diff
@@ -41,6 +41,7 @@ jobs:
             doesn't start with an uppercase character.
           types: |
             break
+            security
```

Comment:
suggestion: not done by this PR but break shouldn't be a type, thats done via exclamation mark, so we should remove this while here.

I also suggest we also add 'ci' (for continuous integration, like GitHub action changes), as a type as users aren't typically interested in those so useful to categorise them separately.

Reply:
I usually add CI work under `chore(ci), chore(gha)`

Reply:
We can do that, will need documenting as `ci` could be what others, like me might naturally use but that should get caught by the validation so as long as we're consistent all good.

Reply:
Yeah, we need documenting that chore would be the umbrella category for build, ci, perf, test, etc, as we don't have separate sections in the changelog

Reply:
Question: how would you handle security? It's not covered by the conventional commits spec, and I always wanted to push security improvements to the changelog

Reply:
I think its fine if you want a separate type for that.